examples/scripts/cartesian_reconstruction.py

```diff
@@ -0,0 +1,341 @@
+# %% [markdown]
+# # Basics of MRpro and Cartesian Reconstructions
+# Here, we are going to have a look at a few basics of MRpro and reconstruct data acquired with a Cartesian sampling
+# pattern.
+# %% [markdown]
+# ## Overview
+# In this notebook, we are going to explore the `mrpro.data.KData` object and the included header parameters.
+# We will then use a FFT-operator in order to reconstruct data acquired with a Cartesian sampling scheme.
+# We will also reconstruct data  acquired on a Cartesian grid but with partial echo and partial Fourier acceleration.
+# Finally, we will reconstruct a Cartesian scan with regular undersampling.
+
+
+# %% tags=["hide-cell"]
+# Get the raw data from zenodo
+import tempfile
+from pathlib import Path
+
+import zenodo_get
+
+dataset = '14173489'
+
+tmp = tempfile.TemporaryDirectory()  # RAII, automatically cleaned up
```

Comment:
the directory including any files in it will be deleted when tmp goes out of scope (also on interpreter shutdown.)
downside is that this will print a warning on shutdown to the console